DataMigration/Output/ElasticSearch/Entity/Product/Model/Product.cs

```diff
@@ -1,40 +1,37 @@
 ﻿using System;
 using System.Collections.Generic;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace DataMigration.Output.ElasticSearch.Entity.Product.Model
```

Comment:
As we established it would be smart to move Product, ProductBase and Variant to separate file. It's going to be bigger and bigger. It can become unreadable.